src/components/AuthorViewpoint/ViewpointCard.tsx

```diff
@@ -0,0 +1,138 @@
+"use client";
+import { TrashIcon, PlusIcon } from "@heroicons/react/24/outline";
+import { Button, TextInput } from "@mantine/core";
+import { useState, useRef, useEffect } from "react";
+import { toast, ToastContainer } from "react-toastify";
```

Comment:
Please use the `sonner` package as that's the toast package we'll be going with in this project.